src/views/Change.vue

```diff
@@ -55,6 +55,23 @@
           sectionNumber="3."
           :validate="getAppValidate"
         />
+        <template v-if="isRoleStaff">
+          <CourtOrderPoa
+            class="'mt-10"
+            sectionNumber="4."
+            :autoValidation="getAppValidate"
+            :draftCourtOrderNumber="getFileNumber"
+            :hasDraftPlanOfArrangement="getHasPlanOfArrangement"
+            :invalidSection="invalidCourtOrder"
```

Comment:
You're passing in 4 props but the component only accepts 1 prop.

Actually, I would move these getters into the component (except getAppValidate).

Reply:
Okay will do I only need to keep getAppValidate as it relies on user input correct?

Reply:
Yes, leave the `getAppValidate` getter since it's at the "same level" as this component. Besides, most of the other components already use it :)

By "same level" I'm thinking of a hierarchical architecture where components only need to know about things at their level of functionality. They don't need to know about what's happening above them or below them (except for formal parameters/properties passed in/out of them).